CMakeLists.txt

```diff
@@ -83,7 +83,7 @@ set(QUIC_TLS_SECRETS_SUPPORT "ON")
 
 configure_file(
     ${CMAKE_CURRENT_SOURCE_DIR}/include/quicer_vsn.hrl.in
```

Comment:
the .hrl.in file should not be in `include` dir, maybe put it in another directory which is not going to be packed into a hex package.